nautobot/extras/models/__init__.py

```diff
@@ -23,20 +25,22 @@
     "ConfigContext",
     "ConfigContextModel",
     "ConfigContextSchema",
-    "Status",
-    "StatusField",
-    "StatusModel",
     "CustomField",
     "CustomFieldChoice",
     "CustomFieldModel",
     "CustomLink",
     "ExportTemplate",
+    "FileAttachment",
+    "FileProxy",
     "GitRepository",
     "GraphQLQuery",
     "ImageAttachment",
     "Job",
     "JobResult",
     "ObjectChange",
+    "Status",
+    "StatusField",
+    "StatusModel",
```

Comment:
`Status*` probably belongs after `Relationship*`, as long as you're moving it around anyway. :-)